ckanext/bcgov/util/util.py

```diff
@@ -26,6 +26,7 @@
 site_url = config.get('ckan.site_url')
 api_key = config.get('ckan.api_key')
 log = logging.getLogger('ckanext.bcgov.util')
+download_resource_id = config.get('ckan.datasource.downloads.resource')
```

Comment:
Move this line into the get_package_tracking function

Reply:
done.